dali/pipeline/executor/executor.h

```diff
@@ -37,7 +40,7 @@ using ExecutorMetaMap = std::unordered_map<std::string, std::vector<ExecutorMeta
 class DLL_PUBLIC ExecutorBase {
  public:
   DLL_PUBLIC virtual ~ExecutorBase() {}
-  DLL_PUBLIC virtual void Build(OpGraph *graph, vector<string> output_names) = 0;
+  DLL_PUBLIC virtual void Build(OpGraph *graph, vector<string> outputs) = 0;
```

Comment:
Why renaming the `output_names` to `outputs`? It's still `output_names` in te dervied `execture_impl`'s `Executor`.

Reply:
That's accidental - I removed this function and then re-added it manually from memory.